src/driver/spanner/SpannerConnectionOptions.ts

```diff
@@ -146,4 +147,6 @@ export interface SpannerConnectionOptions
     }
 
     readonly poolSize?: never
+
+    readonly sessionPool?: SpannerSessionPoolOptions
 }
```

Comment:
How this new `sessionPool` would collide with the `poolSize` option there? Wondering if this would collide with the `sessionPool.max`?